integration/tests/params.go

```diff
@@ -245,7 +247,7 @@ func (p *Params) attemptSettingUserID() {
 		return
 	}
 
-	out, err := clusterProvider.Provider.STS().GetCallerIdentity(&sts.GetCallerIdentityInput{})
+	out, err := clusterProvider.Provider.STSV2().GetCallerIdentity(context.TODO(), &sts.GetCallerIdentityInput{})
```

Comment:
can we have the context be passed down from the `ctl/` file? It can still be a `TODO` but having it passed down from the top ensures we re-use it across the whole command

Reply:
There's no `ctl` here since this is the integration tests, but I have added it now for eksctl commands.